portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,92 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+     <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+
+   <div class="w3-sidebar w3-bar-block w3-border-right" style="display:none" id="mySidebar">
```

Comment:
Indentation seems off here, and other places in the file. Are you using consistent indentation?

Reply:
done